src/libuvc/libuvc.h

```diff
@@ -17,65 +17,65 @@ struct libusb_device_handle;
  * @ingroup diag
  */
 typedef enum uvc_error {
```

Comment:
Did you replace tab with white spaces ?  I cannot see it in the diff tool

Reply:
Not that I am aware of